string first_name
string last_name
uint32 age
float64 gpa
string major
bool is_undergraduate
